From fcfa7ae3a6bb073cdf6e640b0585e916683dd807 Mon Sep 17 00:00:00 2001
From: Hec <hec@heccraft.com>
Date: Thu, 28 May 2026 01:06:58 -0400
Subject: [PATCH 3/5] vulkan-tools: build without /app prefix

---
 elements/extensions/vulkaninfo/vulkan-tools.bst | 1 -
 1 file changed, 1 deletion(-)

diff --git a/elements/extensions/vulkaninfo/vulkan-tools.bst b/elements/extensions/vulkaninfo/vulkan-tools.bst
index b948b692e..ac6fe546d 100644
--- a/elements/extensions/vulkaninfo/vulkan-tools.bst
+++ b/elements/extensions/vulkaninfo/vulkan-tools.bst
@@ -1,7 +1,6 @@
 kind: cmake
 
 (@):
-- elements/extensions/include/app.yml
 - elements/include/vulkan-stack.yml
 
 build-depends:
-- 
2.54.0

